frontend/src/lib/instance-views/elements/VStack.svelte

```diff
@@ -27,13 +26,15 @@
 	<Error type="Incorrect Data" message={errorMessage} />
 {:else}
 	<div class="flex flex-col">
-		{#each keys as key}
-			{#if spec.keys && spec.keys[key] !== undefined && spec.keys[key].type !== undefined}
+		{#each Object.keys(spec.keys) as key}
+			{#if key in jsonData}
```

Comment:
I don't think it makes sense to show them in the order of the spec but rather in the order they are in the json data. This makes it more flexible and views can be combined more freely.